62.CAD/main.cpp

```diff
@@ -1665,13 +1863,48 @@ class CADApp : public ApplicationBase
 		cb->reset(video::IGPUCommandBuffer::ERF_RELEASE_RESOURCES_BIT); // TODO: Begin doesn't release the resources in the command pool, meaning the old swapchains never get dropped
 		cb->begin(video::IGPUCommandBuffer::EU_ONE_TIME_SUBMIT_BIT); // TODO: Reset Frame's CommandPool
 		cb->beginDebugMarker("Frame");
+
+		
+		double theta = 0.0;// (timeElapsed * 0.00008)* (2.0 * nbl::core::PI<double>());
+
+		auto rotation = float64_t3x3(
+			cos(theta), -sin(theta), 0.0,
+			sin(theta), cos(theta), 1.0,
+			0.0, 0.0, 1.0
+		);
+
```

Comment:
you can apply this rotation test only when your test case is active by ` if constexpr `